From 6167666f43da361b2a1eda0a14d42c5b8cb9ae0d Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Tue, 30 May 2023 13:12:35 +0200
Subject: [PATCH] Disable running gyp on shared deps

Signed-off-by: rpm-build <rpm-build>
---
 node.gyp | 17 -----------------
 1 files changed, 18 deletions(-)

diff --git a/node.gyp b/node.gyp
index 33bc7d9..a216c41 100644
--- a/node.gyp
+++ b/node.gyp
@@ -797,23 +797,6 @@
               ],
             },
           ],
-         }, {
-           'variables': {
-              'opensslconfig_internal': '<(obj_dir)/deps/openssl/openssl.cnf',
-              'opensslconfig': './deps/openssl/nodejs-openssl.cnf',
-           },
-           'actions': [
-             {
-               'action_name': 'reset_openssl_cnf',
-               'inputs': [ '<(opensslconfig)', ],
-               'outputs': [ '<(opensslconfig_internal)', ],
-               'action': [
-                 '<(python)', 'tools/copyfile.py',
-                 '<(opensslconfig)',
-                 '<(opensslconfig_internal)',
-               ],
-             },
-           ],
          }],
       ],
     }, # node_core_target_name
-- 
2.49.0

